kukur/auth.py

```diff
@@ -0,0 +1,51 @@
+"""Kukur module for authentication and authorization."""
+
+import time
+from dataclasses import dataclass
+
+import requests
+from requests.auth import AuthBase
```

Comment:
The `try` - `except` should be around this line. The import `from kukur.auth import OIDCConfig` would still fail without it.